.travis.yml

```diff
@@ -10,6 +10,16 @@ matrix:
     env: DJANGO=1.9
   - python: 3.4 # Django 2 support
     env: DJANGO==2.0
+  - python: 3.5 
+    env: DJANGO==2.0
+  - python: 3.6
+    env: DJANGO==2.0
+  - python: 3.4 
+    env: DJANGO==2.2.1
+  - python: 3.5 
+    env: DJANGO==2.2.1
+  - python: 3.6
+    env: DJANGO==2.2.1
```

Comment:
I think that it worth to use `~=2.0` instead of `==2.0` and `~=2.2` instead of `==2.2.1`.
https://www.python.org/dev/peps/pep-0440/#compatible-release